packages/api/src/external/carequality/organization-template.ts

```diff
@@ -0,0 +1,211 @@
+        <type value="both"/>
+        <line value="${addressLine1}"/>
+        <city value="${city}"/>
+        <postalCode value="${postalCode}"/>
+        <country value="USA"/>
+    </address>
+</contact>
+<!--General organizational physical and mailing address-->
+<address>
+    <use value="work"/>
+    <type value="both"/>
+    <line value="${addressLine1}"/>
+    <city value="${city}"/>
+    <state value="${state}"/>
+    <postalCode value="${postalCode}"/>
+    <country value="USA"/>
+    <extension url="OrgPosition">
+        <valueCodeableConcept>
+            <coding>
+                <system value="https://sequoiaproject.org/StructureDefinition/Address/Position/1.0.0"/>
+                <value>
+                    <position>
+                        <longitude value="${longitude}"/>
+                        <latitude value="${latitude}"/>
+                    </position>
+                </value>
+            </coding>
+        </valueCodeableConcept>
+    </extension>
+</address>
+<!--If this is a Carequality Connection, then the partOf object points to the Carequality Implementer-->
+<!--If this is a eHealth Exchange Sub-Participant, then the partOf object points to the eHealth Exchange Participant-->
+<!--If this is a lower-level organization, then the partOf object points to the immediate parent organization-->
+<!--If this is a top-level organization, then the partOf object is not specified-->
+<!--The Endpoints are represented as contained resources and are repeated as needed-->
+${endpointXCPD}
+${endpointDQ}
+${endpointDR}
+</Organization>`;
+}
+
+function getEndpoint(
+  oid: string,
+  url: string,
+  urlType: string,
+  channelType: string,
+  channelName: string
+) {
+  return `<contained>
+    <Endpoint>
+        <!--Home Community ID-->
+        <identifier>
+            <use value="official"/>
+            <type value="HCID"/>
+            <system value="http://www.hl7.org/oid/"/>
+            <value value="${oid}"/>
+        </identifier>
+        <!--Assigning Authority ID-->
+        <identifier>
+            <use value="official"/>
+            <type value="AAID"/>
+            <system value="http://www.hl7.org/oid/"/>
+        </identifier>
+        <connectionType>
+            <system value="http://hl7.org/fhir/subscription-channel-type"/>
+            <code value="${channelType}"/>
+        </connectionType>
+        <name value="${channelName}"/>
+        <managingOrganization/>
+        <address value="${url}"/>
+        <payloadMimeType value="application/json"/>
+        <payloadType>
+            <coding>
+                <system value="urn:oid:1.3.6.1.4.1.19376.1.2.3"/>
+            </coding>
+        </payloadType>
+        <extension url="https://sequoiaproject.org/StructureDefinition/Endpoint/main/1.0.0">
+            <extension url="Transaction">
+                <valueString value="${urlType}"/>
+            </extension>
+            <!--Is this gateway an initiator or responder actor-->
```

Comment:
not a huge deal, but if we don't need these comments while sending messages, I suggest we remove them - machine-to-machine comms should be as lean as possible, this will build up over time.